test/source_file_matcher.ts

```diff
@@ -0,0 +1,74 @@
+import MatchersUtil = jasmine.MatchersUtil;
```

Comment:
let's rename as testing/source_file_matcher.ts 

usually testing/xxx means helper or util for tests.
test/xxx means test cases.